### DISCUSSION TITLE
How to use the cropped ytbb dataset from the BaiduNetDisk link?

### DISCUSSION THREAD

**Discussion by xiaofeinot** [STVIR/pysot#412] [General] [answered]
Form "pysot/training_dataset/yt_bb/readme.md", I can only find how to use the raw ytbb dataset. But I can't find the way to process the cropped dataset from the BaiduNetDisk link. Will this be updated? Or anyone knows how to process the cropped dataset? Thanks very much.

**Comment by whjzsy** [comment]
I have the same problem?Do you solve it?

**Comment by JayJayBinks1** [comment, marked as answer]
You shouldn't need to process it. From memory, I think you can just place them in training_dataset/yt_bb/crop511 and it should work.

**Reply by xyl-507** [reply]
Sorry to bother you. It's my first contact with SOT.when I use yt_bb to train ,there is a error:
`AttributeError:'NoneType' object has no attribute 'shape'` . 
Can you give me some advice? Thanks in advance
